src/System.Management.Automation/engine/CommandDiscovery.cs

```diff
@@ -1502,7 +1520,24 @@ internal IEnumerator<CmdletInfo> GetCmdletInfo(string cmdletName, bool searchAll
                 List<CmdletInfo> cmdlets;
                 if (!scope.CmdletTable.TryGetValue(commandName.ShortName, out cmdlets))
                 {
-                    continue;
+                    if (useAbbreviationExpansion)
+                    {
+                        foreach (List<CmdletInfo> cmdletList in scope.CmdletTable.Values)
+                        {
+                            foreach (CmdletInfo cmdletInfo in cmdletList)
+                            {
+                                string abbreviatedCmdlet = new string(cmdletInfo.Name.Where(c => char.IsUpper(c)).ToArray());
```

Comment:
Do we care about `-` here?

Reply:
Actually, this code is no longer needed since I'm treating `-UseAbbreviationExpansion` as a command name pattern so the code will never call this method when `-UseAbbreviationExpansion` is used.  I'll clean this up.